src/content/presenters/ConsoleFramePresenter.ts

```diff
@@ -19,10 +19,13 @@ export class ConsoleFramePresenterImpl implements ConsoleFramePresenter {
       return;
     }
 
+    /* @ts-ignore */
+    const colorScheme = getComputedStyle(document.getElementsByTagName('body')[0]).colorScheme
```

Comment:
Why not `document.body` over `document.getElementsByTagName("body")`? What of the `null` / `undefined` situation?

Reply:
Good call, thanks!